Json-Rpc/Handler.cs

```diff
@@ -534,10 +517,40 @@ private JsonRpcException PreProcess(JsonRequest request, object context)
             return externalPreProcessingHandler(request, context);
         }
 
+        private JsonResponse PostProcess(Action<JsonResponse> callback, JsonRequest request, JsonResponse response, object context)
+        {
+            if (externalPostProcessingHandler != null)
+            {
+                try
+                {
+                    JsonRpcException exception = externalPostProcessingHandler(request, response, context);
+                    if (exception != null)
+                    {
+                        System.Diagnostics.StackTrace t = new System.Diagnostics.StackTrace();
```

Comment:
Hi sorry, no this is some leftover debug where i had issues with some code paths not running as i expected so i was tracing the origin, i'll remove t from here, thanks for catching this.